.github/workflows/ci.yml

```diff
@@ -42,18 +42,19 @@ jobs:
         with:
           path: dist/*
 
-  pytest-python:
-    name: PyTest
+  pytest-django-lts:
```

Comment:
Nope, that's not how this works.
One is to test all Python versions, the other for all Django versions.

Reply:
Ah, now I get it. Thanks 👍